Source/WebCore/page/ios/ContentChangeObserver.cpp

```diff
@@ -223,6 +223,18 @@ void ContentChangeObserver::completeDurationBasedContentObservation()
     adjustObservedState(Event::EndedFixedObservationTimeWindow);
 }
 
+bool ContentChangeObserver::isObservedPropertyForTransition(AnimatableProperty property) const
```

Comment:
Is this used outside this class? If not, file static?

Reply:
Fixed in updated commit.